src/main/io/gps.c

```diff
@@ -686,77 +1100,144 @@ void gpsInitUblox(void)
                 break;
             }
 
+            DEBUG_SET(DEBUG_GPS_UNIT_CONNECTION, 5, gpsData.state_position);
+
             if (gpsData.ackState == UBLOX_ACK_IDLE) {
                 switch (gpsData.state_position) {
+                    // if a UBX command is sent, ack is supposed to give position++ once the reply happens
+                    case UBLOX_DETECT_UNIT:
+                        if (gpsData.unitVersion == UBX_VERSION_UNDEF) {
+                            ubloxSendClassMessage(CLASS_MON, MSG_MON_VER, 0);
+                            break;
+                        }
+                        gpsData.state_position++;
+                        break;
                     case UBLOX_INITIALIZE:
-                        gpsData.ubloxUsePVT = true;
-                        gpsData.ubloxUseSAT = true;
+                        ubloxSetNavRate(1, 1, 1); // throttle nav data rate to once per second, until configured
+                        break;
+                    case UBLOX_ACQUIRE_MODEL:
                         ubloxSendNAV5Message(gpsConfig()->gps_ublox_acquire_model);
+                        gpsData.ubloxUsingFlightModel = false;
+                        break;
+                    case UBLOX_CFG_ANA:
+                        if (gpsData.ubloxM7orAbove) { // NavX5 support existed in M5 - in theory we could remove that check
+                            ubloxSendNavX5Message();
+                        } else {
+                            gpsData.state_position++;
```

Comment:
This needs an explanation (across the file): why do we turn `CFG_ANA` into a no-op?

Reply:
I agree that AssistNow Autonomous support should be developed in a separate PR, meaning that it should be removed from this PR.  It's Freasy's work, and he is busy right now, so now the code will be commented out at this point, and likely removed later.

Reply:
The assist now autonomous code will be commented out and ultimately should go into a separate PR.